### PR TITLE
Add Physical Measurements configs to aouRT underlay

### PR DESCRIPTION
**Pull request by dolbeew**
Add entities for blood pressure, bmi, heart rate, height, hip circumference, waist circumference and weight.

Screenshot 2024-02-06 at 11 37 55 PM

### CODE REVIEW

File: underlay/src/main/resources/config/datamapping/aouRT/entity/bloodPressureOccurrence/all.sql
Comment (chenchals) [inline underlay/src/main/resources/config/datamapping/aouRT/entity/bloodPressureOccurrence/all.sql:1]: Great!

**Review by chenchals — CHANGES_REQUESTED** [review]
Need the following changes:
- I did not see indexed tables for `SC2023Q3R2` dataset for physical measurements 
  -  underlay/src/main/resources/config/underlay/aouSC2023Q3R2/ui.json
  - underlay/src/main/resources/config/underlay/aouSR2023Q3R2/underlay.json
- Run indexing build for Controlled tier also 
- I am guessing, you have deployed to local docker in AoU to check the UI

File: underlay/src/main/resources/config/datamapping/aouRT/entity/bloodPressureOccurrence/entity.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "bloodPressure",
```
Comment (chenchals) [inline underlay/src/main/resources/config/datamapping/aouRT/entity/bloodPressureOccurrence/entity.json:2]: change to
"name": "bloodPressureOccurrence",

**Review by freemabd — APPROVED** [review]
Looks good. Just a few comments on folder names.

**Review by chenchals — CHANGES_REQUESTED** [review]
Need to have changes to be added/modified for physical measurements
- underlay/aouSC2023Q3R2/ui.json
- underlay/aouSC2023Q3R2/underlay.json

**Comment by dolbeew** [comment]
I renamed all entity folders to drop 'Occurrence.' Also added all entities and groups to the aouCT underlay and tested locally in the Workbench after running indexing.

**Review by chenchals — APPROVED** [review]
Thank you for the changes!

**Pull request merged**